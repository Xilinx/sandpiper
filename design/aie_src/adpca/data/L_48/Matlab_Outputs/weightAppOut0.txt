5.443263799e-03 -8.993653581e-03
1.184091282e+01 -1.707618713e+01
1.085710907e+01 4.268420219e+00
