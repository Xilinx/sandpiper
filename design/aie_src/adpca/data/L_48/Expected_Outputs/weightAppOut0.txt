5.443225615e-03 -8.993584663e-03 
1.184047508e+01 -1.707545280e+01 
1.085674667e+01 4.268432140e+00 
